### ISSUE TITLE
Some questions about training and inferencing 

### ISSUE THREAD

**Issue by godspirit00** [alibaba-damo-academy/KAN-TTS#25]
Hello, I have some questions.
1. If I want to train a model on the Nancy Corpus (blizzard challenge 2011 dataset), how can I prepare the data? Is there an example of the "general data" mentioned in the wiki?
2. Is it possible to control the speed of the speech when inferencing?
Thank you!

**Comment by GinChow** [comment]
Hi @godspirit00 , 
1. Here is an example dataset https://modelscope.cn/datasets/modelscope/DAMO.NLS.KAN-TTS.OpenDataset/summary, and we will release some other tts datasets on https://modelscope.cn/datasets soon
2. Currently there is no explicit argument to control the speech speed, but I think you can try scale the duration in VarianceAdaptor 
https://github.com/alibaba-damo-academy/KAN-TTS/blob/2f54c741ab9bfc890f27a1950918be173c836848/kantts/models/sambert/adaptors.py#L9-L24

```python
class LengthRegulator(nn.Module):
    def __init__(self, r=1):
        super(LengthRegulator, self).__init__()

        self.r = r

    def forward(self, inputs, durations, masks=None):
        scale = 1.2 
        reps = (durations*scale + 0.5).long()
        output_lens = reps.sum(dim=1)
        max_len = output_lens.max()
```

**Comment by godspirit00** [comment]
Hi @GinChow , thanks for the reply.
1. I see in the example dataset in the `prosody.txt` that there are additional labels in the transcription besides the text itself. Is there an explanation of each kind of label? How can I turn existing plain text into such transcription?
Thank you!

**Comment by GinChow** [comment]
> 

1. The labeling standard will be released recently, and so will the automatic labeling service.
2. Sure, thanks for your feedback :)
